plugins/janus_videoroom.c

```diff
@@ -1772,6 +1680,7 @@ static guint32 janus_videoroom_rtp_forwarder_add_helper(janus_videoroom_publishe
 	forward->ssrc = ssrc;
 	forward->substream = substream;
 	forward->is_data = is_data;
+	// host to ip
```

Comment:
Please remove this comment as well (which again I assume is a leftover while studying the code?)